types/react-loadable-visibility/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "@loadable/component": "^5.10.1"
```

Comment:
this points to the original `@loadable/component` which is untyped. You should delete all of package.json since you only need to depend on packages within DT.

Reply:
Done. Does it make sense to revert the PR that added `@loadable/component` to the whitelist?
https://github.com/microsoft/types-publisher/pull/651